gui/ajax.php

```diff
@@ -262,7 +262,12 @@ function ajaxCheckProgress(){
 	$x = 0;
 	foreach ($active_sims as $row) {
 		$iteration = $row['iteration'];
-		$status = substr($row['status'],-4,2);
+		if($row['status']=='1100'){
+			$status = substr($row['status'],-3,3);
+		}
+		else{
+			$status = substr($row['status'],-2);
+		}
 		if ($status == "00") { $status="100";}
```

Comment:
Can this line be reached at all since we catch 1100 status before?

Reply:
You're right.